src/Components/Workspace/index.js

```diff
@@ -283,6 +283,55 @@ const WorkspaceView = props => {
 
   // general
 
+  const formatDuration = async (duration) => {
```

Comment:
woohoo, this is looking so good now 🤠 

just thinking - maybe we can move this function to the Utils directory? this file is getting huuuge 🙈

Reply:
Good shout!